core/platform/search/elastic_search_services.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides platform search services implemented using the elastic search python
+API.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import json
+
+import elasticsearch
+import feconf
+import python_utils
+
+MAXIMUM_NUMBER_OF_PAGES = 10000
```

Comment:
Is this the max number of pages, or the max number of results per page, or the max number of docs in the index?

Reply:
Maximum number of total pages to load for searching. I've clarified in the comments.